packages/shared/src/contexts/BootProvider.tsx

```diff
@@ -292,12 +293,14 @@ export const BootDataProvider = ({
               getPage={getPage}
               deviceId={deviceId}
             >
-              <NotificationsContextProvider
-                isNotificationsReady={isBootReady}
-                unreadCount={notifications?.unreadNotificationsCount}
-              >
-                {children}
-              </NotificationsContextProvider>
+              <ErrorBoundary>
```

Comment:
Lowest I can put it because log context needs other contexts, but works if I throw from layout